elasticsearch/tests/elasticsearch_test.py

```diff
@@ -3,6 +3,7 @@
 sys.path.insert(1, os.path.join(sys.path[0], '../../helpers'))
 from helpers import helm_template
 import yaml
+import json
```

Comment:
Did you mean to add this import? I don't see it being used anywhere.

Reply:
No it should not have been there, I used that to output a bunch of stuff when I started looking into the tests. 

Removed in [e7f76dc](https://github.com/elastic/helm-charts/pull/265/commits/e7f76dce3a69c451ff72830cbef57e62631663e1).